test/conformance/gateway_conformance_test.go

```diff
@@ -29,14 +31,27 @@ var (
 )
 
 func TestGatewayConformance(t *testing.T) {
-	t.Parallel()
-
 	t.Log("configuring environment for gateway conformance tests")
 	client, err := client.New(env.Cluster().Config(), client.Options{})
 	require.NoError(t, err)
 	require.NoError(t, gatewayv1alpha2.AddToScheme(client.Scheme()))
 	require.NoError(t, gatewayv1beta1.AddToScheme(client.Scheme()))
 
+	t.Log("starting the controller manager")
+	args := []string{
+		fmt.Sprintf("--ingress-class=%s", ingressClass),
+		fmt.Sprintf("--admission-webhook-cert=%s", testutils.KongSystemServiceCert),
+		fmt.Sprintf("--admission-webhook-key=%s", testutils.KongSystemServiceKey),
+		fmt.Sprintf("--admission-webhook-listen=%s:%d", testutils.AdmissionWebhookListenHost, testutils.AdmissionWebhookListenPort),
+		"--profiling",
+		"--dump-config",
+		"--log-level=trace",
+		"--debug-log-reduce-redundancy",
+		"--feature-gates=GatewayAlpha=true",
+		"--anonymous-reports=false",
+	}
+	require.NoError(t, testutils.DeployControllerManagerForCluster(ctx, env.Cluster(), args...))
```

Comment:
Given we only have one test case in this package, wouldn't it make sense to drop `TestMain` and move the whole setup to `TestGatewayConformance`?

Reply:
That might be a valid approach as well. But OTOH to keep it consistent we can keep `TestMain` to handle the cluster creation/deletion (or re-use) just like elsewhere. WDYT?

Reply:
One drawback I see is that if somebody would like to add a test case to this package, it might be misleading that the `TestMain` exists (a false feeling that the whole controller setup is already done). Which probably won't happen. I'm I'm ok with leaving this as it is.